pennylane/operation.py

```diff
@@ -543,7 +543,7 @@ def get_parameter_shift(self, idx, shift=np.pi / 2):
             idx (int): parameter index
 
         Returns:
-            float, float: multiplier, shift
+            list[[float, float, float]]: list of multiplier, coefficient, shift for each execution
```

Comment:
```suggestion
            list[[float, float, float]]: list of multiplier, coefficient, shift for each term in the gradient recipe
```
(based on the docstring of `grad_recipe` above? not 100% sure)